frontend/components/DrawerLogo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+interface IconInterface {
```

Comment:
Keep always in mind the convention for naming props type, namely if your component name is `DrawerLogo`, your props should be `DrawerLogoProps`